.github/workflows/sca.yml

```diff
@@ -72,7 +72,7 @@ jobs:
         run: composer php-compatibility
 
       - name: Check - composer-unused
-        run: ./dev-tools/vendor/bin/composer-unused --no-progress --excludePackage=composer/xdebug-handler
+        run: ./dev-tools/vendor/bin/composer-unused --excludePackage=composer/xdebug-handler --excludePackage=symfony/polyfill-php84
```

Comment:
can you share why this is needed for 84 polyfill but not for previous polyfills ?

Reply:
No, I haven't figured it out.

Reply:
and I assume it's failing without this? :(

Reply:
yes